warehouse/query-core/src/test/java/datawave/query/jexl/functions/ContentFunctionQueryTest.java

```diff
@@ -496,12 +496,16 @@ public boolean isTermFrequencyField(String field) {
         @Override
         public boolean isIndexListField(String field) {
             return false;
-        };
+        }
+        
```

Comment:
Fix spacing before semi-colons?

Reply:
That semicolon was already there - not sure why...  The autoformatter did that, but I have since taken the unnecessary semicolon out.